test/registry-test.ts

```diff
@@ -11,7 +11,7 @@ if (process.platform === 'win32') {
 
       const programFilesDir = values.find(v => v.name == 'ProgramFilesDir')
       expect(programFilesDir!.type).equals('REG_SZ')
-      expect(programFilesDir!.data).contains('C:\\Program Files')
+      expect(programFilesDir!.data).contains(':\\Program Files')
```

Comment:
This change is because you've got Windows installed on a different drive, right?

Reply:
@shiftkey yep.